lib/request-id/decorateLogger.js

```diff
@@ -0,0 +1,73 @@
+/*
+ JBoss, Home of Professional Open Source
+ Copyright Red Hat, Inc., and individual contributors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+var cls = require('continuation-local-storage');
+var namespace = require('../const').namespace;
+
+function getRequestId() {
+  var ns = cls.getNamespace(namespace);
+  if (!ns) {
+    // namespace not initialized, outside of request chain
```

Comment:
You mean adding an specific field to the logs when there's no request id?

I think like you said that there are logging statements unrelated to http requests, and since bunyan's output is JSON,`{}.reqId === {noreqid:true}.reqId === undefined`, and we can still do filtering by `bunyan -c '!this.reqId'`, so I don't think that's necessary.